sheeprl/algos/ppo/ppo_decoupled.py

```diff
@@ -454,21 +454,26 @@ def main():
 
     world_collective = TorchCollective()
     player_trainer_collective = TorchCollective()
-    world_collective.setup(backend="nccl" if os.environ.get("LT_ACCELERATOR", None) in ("gpu", "cuda") else "gloo")
+    world_collective.setup(
+        backend="nccl" if os.environ.get("LT_ACCELERATOR", None) in ("gpu", "cuda") else "gloo",
+        timeout=timedelta(days=1),
+    )
 
     # Create a global group, assigning it to the collective: used by the player to exchange
     # collected experiences with the trainers
-    world_collective.create_group()
+    world_collective.create_group(timeout=timedelta(days=1))
```

Comment:
why is there a timeout here?

Reply:
Thats because on windows the init_process_group can raise a SocketTimeoutError, but setting a high timeout seems to alleviate the problem. The problem is that we can only have control over the timeout when we create the process group or when we can specify the strategy: if the script is run by the Lightning CLI then we don't have control.
Related: [#85621](pytorch/pytorch#85621)